jdaviz/components/plugin_section_header.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div :class="active ? 'strike strike-active text--secondary': 'strike text--secondary'">
+  <div :class="[
+        'strike',
+        active ? ($vuetify.theme.dark ? 'strike-active-dark' : 'strike-active-light') : ''
+    ]">
```

Comment:
does this still need `text--secondary` or does that not add anything necessary?

Reply:
Yes, It doesn't add anything I can see.  Do you recall why you removed it 🙏 , as I built the styles on top of your last changes to the section header?